### ISSUE TITLE
TV Control API also allows control of radio tuners

### ISSUE THREAD

**Issue by tidoust** [w3c/media-web-roadmap#4]
The [Media Control](https://w3c.github.io/media-web-roadmap/media/control.html) page mentions "Radio Tuner Control" as a feature not covered by ongoing work. Despite the name, the TV Control API actually already supports radio tuners (at least in theory) and the TV Control WG explores updates to the API that may still be needed to meet additional requirements that radio might bring (including possibly renaming the interfaces to drop the confusing "TV" prefix).

In short, I think "Radio Tuner Control" should be removed from that page.
